scalding-beam/src/main/scala/com/twitter/scalding/beam_backend/KryoCoder.scala

```diff
@@ -33,10 +36,22 @@ case class OrderedSerializationCoder[T](ordSer: OrderedSerialization[T]) extends
   override def encode(value: T, outStream: OutputStream): Unit = ordSer.write(outStream, value)
```

Comment:
I just noticed an issue here. This should be: `ordSer.write(outStream, value).get` since the method returns a `Try[Unit]` and we want to see that exception. The current code will be hiding an exception if we overrun a buffer, for instance.